storage-operator/cmd/manager/main.go

```diff
@@ -80,14 +83,28 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
+	ctx := context.TODO()
+
+	// Set default manager options
+	options := manager.Options{
 		Namespace:               namespace,
```

Comment:
> Note: because of bff55c2, the 'standard' leader election mechanism/code
is *not* used.

Now that I think about it, this was fixed so we can probably come back to the standard election mechanism.
See https://github.com/operator-framework/operator-sdk/issues/1305

Reply:
Maybe in some other PR at some point :) FWIW, a Kubebuilder-scaffolded operator uses this mechanism (`LeaderElection` in the manager's `Options`) by default.